data-source/json/test_radio_detail_answer_visible.json

```diff
@@ -0,0 +1,128 @@
+{
+	"mime_type": "application/json/ons/eq",
+	"schema_version": "0.0.1",
+	"data_version": "0.0.3",
+	"survey_id": "0",
+	"title": "Radio Visible",
+	"theme": "default",
+	"metadata": [{
+			"name": "user_id",
+			"type": "string"
+		},
+		{
+			"name": "period_id",
+			"type": "string"
+		},
+		{
+			"name": "ru_name",
+			"type": "string"
+		}
+	],
+	"sections": [{
+		"id": "section-1",
+		"groups": [{
+			"id": "radio",
+			"title": "Radio Open",
+			"blocks": [{
+					"type": "Question",
+					"id": "radio-visible-true",
+					"question": {
+						"type": "General",
+						"id": "radio-visible-true-question",
+						"title": "What is your favourite drink?",
+						"answers": [{
+							"type": "Radio",
+							"id": "radio-visible-true-answer",
+							"mandatory": false,
+							"voluntary": true,
+							"options": [{
+									"label": "Coffee",
+									"value": "Coffee"
+								},
+								{
+									"description": "Enter answer",
```

Comment:
Minor, but this description looks odd in the page and should be removed.

Reply:
Removed, thanks!